### ISSUE TITLE
Get result of extrinsics balance transfer using block's extrinsics

### ISSUE THREAD

**Issue by kunalransing** [polkadot-js/api#4923]
Hi,
I'm new to polkadot. I am working on checking received transactions to my wallets.
I have used below code & able to get hash, from, to & amount etc. but I am not able to get result & fee of received balance transfer in **ex** in below code
```
   const signedBlock = await api.rpc.chain.getBlock(blockHash);
    signedBlock.block.extrinsics.forEach((**ex**, index) => {
        if(ex.method.section === 'balances' && ex.method.method === 'transfer'){
            console.log(`Tx[${ex.hash.toString()}] : Amount[${ex.args[1].toString()}] sent from[${ex.signer.toString()}] to[${ex.args[0].toString()}]`);
        }
   });
```
@jacogr Please help me. Thanks.

**Comment by jacogr** [comment]
I would suggest referring HOWTO queries to https://substrate.stackexchange.com/ - it will be a while before I get to dealing with this. Alternatively, hang tight, should get to this in a couple of weeks when I have a gap to write "other" code. (Swamped with writing actual polkadot-js code, so not much time to write code for others)

**Comment by kunalransing** [comment]
I have tried on https://substrate.stackexchange.com/ but didn't find anything useful..may be I have missed. I have asked here to crosscheck that we really have these values in extrinsics data as earlier I have checked **api.query.system.events** but didn't get tx hash there so read about extrinsics. I almost got all required data except result(success or fail) of that transfer.
I will check again. Thank you for your help.
